### PR TITLE
Duplicated placeholder text when pressing new issue many times

### PR DESCRIPTION
**Pull request by alexr00**
Fixes microsoft/vscode-internalbacklog#1138

### CODE REVIEW

File: src/issues/issueFeatureRegistrar.ts [hunk 1]
```diff
@@ -465,6 +465,10 @@ export class IssueFeatureRegistrar implements vscode.Disposable {
 
 	private async makeNewIssueFile(title?: string, body?: string, assignees?: string[] | undefined) {
 		const bodyPath = vscode.Uri.parse(`${NEW_ISSUE_SCHEME}:/NewIssue.md`);
+		if (vscode.window.visibleTextEditors.filter(editor => editor.document.uri.scheme === NEW_ISSUE_SCHEME).length > 0) {
```
Comment (RMacfarlane) [inline src/issues/issueFeatureRegistrar.ts:468]: tslint warning about shadowed name 'editor' here

File: src/issues/issueFeatureRegistrar.ts [hunk 2]
```diff
@@ -465,6 +465,10 @@ export class IssueFeatureRegistrar implements vscode.Disposable {
 
 	private async makeNewIssueFile(title?: string, body?: string, assignees?: string[] | undefined) {
 		const bodyPath = vscode.Uri.parse(`${NEW_ISSUE_SCHEME}:/NewIssue.md`);
+		if (vscode.window.visibleTextEditors.filter(editor => editor.document.uri.scheme === NEW_ISSUE_SCHEME).length > 0) {
+			return;
+		}
+		await vscode.workspace.fs.delete(bodyPath);
```
Comment (RMacfarlane) [inline src/issues/issueFeatureRegistrar.ts:471]: not sure I understand this line, why is this needed?

Comment (alexr00) [reply src/issues/issueFeatureRegistrar.ts:471]: It seemed like the easiest way to reset the file and get rid of the decorations. If the file is still open in a tab that isn't currently visible, the decorations will still be there and can end up in the wrong place.
